nginx/https_headers.conf

```diff
@@ -0,0 +1,15 @@
+# The following headers were added to pass the mozilla http observatory
+# test suit.
+# https://wiki.mozilla.org/Security/Guidelines/Web_Security
+
+# All of them are very simple in this case, because both homu and buildbot
+# do not use external resources.
+
+# CSP is the most important one
```

Comment:
The options we're setting are fairly basic and IMO the headers are readable enough as is that we don't need an expository comment for each one.